### PR TITLE
fix hitl, make beta ci optional

### PR DESCRIPTION
**Pull request by jordens**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -122,35 +122,3 @@ jobs:
           # Install depedencies at our cache location
           mv ../target/thumbv7em-none-eabihf/doc src/firmware
           mdbook build
-
-  hitl-trigger:
-    runs-on: ubuntu-latest
-    environment: hitl
-    if: ${{ github.event_name == 'merge_group' }}
```
Comment (ryan-summers) [inline .github/workflows/ci.yml:129]: I think this condition here is critical to correct operation of the HITL + merge queue setup. If we don't have this condition, the job isn't marked as "Skipped" on a PR, but rather just never runs. With that setup, we can never actually ad it to the merge queue

**Review by ryan-summers — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,8 +58,10 @@ jobs:
         include:
```
Comment (ryan-summers) [inline .github/workflows/ci.yml:58]:
```suggestion
        continue-on-error: false
        include:
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -49,7 +49,7 @@ jobs:
 
   compile:
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.toolchain == 'nightly' }}
+    continue-on-error: ${{ matrix.continue-on-error }}
```
Comment (ryan-summers) [inline .github/workflows/ci.yml:52]: Maybe 
```suggestion
    continue-on-error: ${{ matrix.continue-on-error || 'false'}}
```

Comment (jordens) [reply .github/workflows/ci.yml:52]: Ah. RTFM.

**Pull request merged**